src/main/java/com/programmers/library/utils/ConsoleIO.java

```diff
@@ -0,0 +1,29 @@
+package com.programmers.library.utils;
+
+import java.util.Scanner;
+
+public class ConsoleIO {
+    private final Scanner scanner;
+
+    public ConsoleIO() {
+        scanner = new Scanner(System.in);
+    }
+
+    public String getStringInput(String message) {   // 문자열 입력
+        printMessage(message);
+        System.out.print("> ");
+        return scanner.nextLine();
+    }
+
+    public int getIntInput(String message) {     // 정수 입력
+        printMessage(message);
+        System.out.print("> ");
+        int input = scanner.nextInt();
+        scanner.nextLine();
+        return input;
+    }
+
+    public void printMessage(String message) {  // 메시지 출력
+        System.out.println(message);
+    }
+}
```

Comment:
```suggestion
public final class ConsoleIO {
    private static final Scanner scanner = new Scanner(System.in);

    private ConsoleIO() {}

    public static String getStringInput(String message) {   // 문자열 입력
        printMessage(message);
        System.out.print("> ");
        return scanner.nextLine();
    }

    public static int getIntInput(String message) {     // 정수 입력
        printMessage(message);
        System.out.print("> ");
        int input = scanner.nextInt();
        scanner.nextLine();
        return input;
    }

    public static void printMessage(String message) {  // 메시지 출력
        System.out.println(message);
    }
}
```

차라리 Util 클래스로 만들어서 필요한 부분에서 사용하는 것이 좋아 보이네요. 
Utils 클래스라는 객체지향적인 설계를 위반하는 클래스가 나오긴 하지만 콘솔 프로그램이라는 특성상 어쩔 수 없는 부분이라 생각합니다.
그럴바엔 그냥 이부분에 대해서 고민하는 것은 포기하고, 그 외에 부분에서 객체지향적인 고민을 하는 것이 좋을 것 같습니다.